examples/function-templates/typescript/contentful-app-manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "functions": [
+    {
+      "id": "example",
+      "name": "Example Function",
+      "description": "This is an example to help you learn how functions work.",
+      "path": "functions/example.js",
+      "entryFile": "functions/example.ts",
+      "allowNetworks": []
```

Comment:
Accepts?

Reply:
This just for renaming. That change will come with a separate PR.